tdrs-frontend/package.json

```diff
@@ -9,9 +9,11 @@
     "axios": "^0.21.1",
     "classnames": "^2.2.6",
     "connected-react-router": "^6.8.0",
+    "file-type": "^16.3.0",
     "history": "^4.7.2",
     "immutable": "^4.0.0-rc.12",
     "include-media": "^1.4.9",
+    "miragejs": "^0.1.41",
```

Comment:
will we need this after #826 is addressed?

Reply:
Yes we hope to keep it around to use whenever we need to stub endpoints in the frontend so that it's never blocked by pending backend work. I believe @riatzukiza is using it too for his Downloads work.